### ISSUE TITLE
Docs Improvement

### ISSUE THREAD

**Issue by PaxPax** [Wolfr/sveltekit-jui#80]
I think it would be helpful to show all components with an option to toggle the source code shown and all props, events, and slots that are applicable.

**Comment by Wolfr** [comment]
Do you mean every component should have source code and a props table? A related issue is #2 .

**Comment by PaxPax** [comment]
yes, along with what events are forwarded and what are dispatched and what values to expect from dispatch.

*edit: I really like how ibm carbon components show it. example https://carbon-svelte.vercel.app/components/TextInput

**Comment by Wolfr** [comment]
I am fully along with doing it this way, and Carbon is a good reference. Now we just need people to help do the work :-)

**Comment by PaxPax** [comment]
so implementing something that would look like 
![image](https://user-images.githubusercontent.com/25893271/114968747-9631bb00-9e3c-11eb-9b09-0c58e4dc5b72.png)
What are your thoughts?

**Comment by Wolfr** [comment]
I think that's a great improvement to have! Especially for highly interactive elements.

BTW all those scroll bars that you see, seems like I have to set `overflow: auto;` instead of scroll. You shouldn't need a horizontal scrollbar on your screen shot.
